editors/sc-ide/widgets/multi_editor.hpp

```diff
@@ -32,6 +32,8 @@
 #include <QTextDocument>
 #include <QSplitter>
 
+#include "../doc_manager.hpp"
+
```

Comment:
just out of curiosity - how do the occasional additional new imports are coming into place here? Are they Qt specific, do they help the compiler/IDE to be more explicit or are these just left-overs from development?

Reply:
Interesting. Looking forward to the Qt6 PR :)

Reply:
Coming up! Once the previous CI passes, so that we have that brew universal cache...